### ISSUE TITLE
mysql-async : What is mysql JSON data type in Scala ?

### ISSUE THREAD

**Issue by boyfox** [mauricio/postgresql-async#208]
Hi @mauricio !

This is a question, I have mysql JSON data type in my "notifications" table and What is type need use for parsing JSON type in mysql-async library. Please help me

**Comment by SeanSWatkins** [comment]
Seems to give a MatchError when using a JSON type, according to the mysql documentation it's just a string once queried from the db. A temp work around is to cast it to a string using the following: 
`CAST(jsonColumnName AS CHAR) as 'json_string'`

Or something of the sort

**Comment by boyfox** [comment]
Yes you are right, problem with parsing. I changed JSON data type to TEXT and solved problem
